packages/katello/rubygem-redhat_access/rubygem-redhat_access.spec

```diff
@@ -0,0 +1,129 @@
+
+%global gem_name redhat_access
+%global plugin_name redhat_access
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.1.1
+Release: 1%{?foremandist}%{?dist}
+Summary: Plugin to add Redhat Access to Foreman
+Group: Applications/Systems
+License: GPLv3+
+URL: https://github.com/redhataccess/foreman-plugin
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: foreman >= 1.18.0
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby 
+Requires: %{?scl_prefix_ruby}ruby(rubygems) 
+Requires: %{?scl_prefix}rubygem(redhat_access_lib) >= 1.1.0
+Requires: %{?scl_prefix}rubygem(angular-rails-templates) >= 0.0.4
+Requires: %{?scl_prefix}rubygem(katello) >= 3.4.0
+BuildRequires: foreman-assets
+BuildRequires: foreman-plugin >= 1.18.0
+BuildRequires: %{?scl_prefix}rubygem(foreman-tasks)
+BuildRequires: %{?scl_prefix}rubygem(redhat_access_lib) >= 1.1.0
+BuildRequires: %{?scl_prefix}rubygem(angular-rails-templates) >= 0.0.4
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby 
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel 
+BuildRequires: %{?scl_prefix}rubygem(redhat_access_lib) >= 1.0.1
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+Provides: foreman-plugin-%{plugin_name}
+
+%description
+This plugin adds Red Hat Access knowledge base search, case management and
+diagnostics to Foreman.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+mkdir -p %{buildroot}%{foreman_bundlerd_dir}
+mkdir -p %{buildroot}/etc/redhat_access
+
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+# Copy config file
+cp -pa $RPM_BUILD_DIR/%{gem_name}-%{version}/config/config.yml.example %{buildroot}/etc/redhat_access/config.yml
+
+%foreman_bundlerd_file
+%foreman_precompile_plugin -s
+
+# Below is static assets hack - here until we figure out how to do precompile properly
+cp -r  $RPM_BUILD_DIR/%{gem_name}-%{version}/vendor/assets/images/*  %{buildroot}/%{gem_dir}/gems/%{gem_name}-%{version}/public/assets
+cp -r  $RPM_BUILD_DIR/%{gem_name}-%{version}/vendor/assets/fonts/*  %{buildroot}/%{gem_dir}/gems/%{gem_name}-%{version}/public/assets
```

Comment:
I'm not exactly sure what this is doing, but assets don't load.

Reply:
oh I see you also tested and found out, I'll continue tomorrow but if you figure out something, let me know, the code seems to use `<%= stylesheet 'insights/application' %>` which is what we use elsewhere too